megalista_dataflow/main.py

```diff
@@ -456,4 +458,4 @@ def run(argv=None):
     logging.getLogger().setLevel(logging.ERROR)
     logging.getLogger("megalista").setLevel(logging.INFO)
     run()
-    logging.getLogger("megalista").info("Completed successfully!")
+    logging.getLogger("megalista").info(f"MEGALISTA build {MEGALISTA_VERSION}: Completed successfully!")
```

Comment:
could this line be moved to line 459 before calling run? That way the Megalista version will always log with; success or errors.